desktop-widgets/tab-widgets/maintab.cpp

```diff
@@ -1124,41 +1124,13 @@ void MainTab::divetype_Changed(int index)
 			  get_dive_dc(current_dive, dc_number)->divemode);
 }
 
-void MainTab::on_watertemp_textChanged(const QString &text)
+void MainTab::on_watertemp_editingFinished()
 {
-	if (editMode == IGNORE || acceptingEdit == true)
+	if (editMode == IGNORE || acceptingEdit == true || !current_dive)
 		return;
-	displayed_dive.watertemp.mkelvin = parseTemperatureToMkelvin(text);
-	markChangedWidget(ui.watertemp);
-	validate_temp_field(ui.watertemp, text);
-}
-
-void MainTab::validate_temp_field(QLineEdit *tempField, const QString &text)
-{
```

Comment:
I'm sure this will be obvious to me in a moment, but can you explain one more time why you no longer want to validate the field?

Reply:
Because the field is rewritten after editing anyway. That does in principle the same as this validate call.